assets/js/src/form_editor/components/header.jsx

```diff
@@ -53,6 +61,22 @@ const Header = ({ isInserterOpened, setIsInserterOpened }) => {
           isPressed={sidebarOpened}
           className="mailpoet-editor-header-button"
         />
+        <DropdownMenu
+          icon={moreVertical}
+          label="More tools & options"
```

Comment:
This should be translated. In case this text is taken from WP post editor you can use `__()`  from @wordpress/i18n since we [load translations for post editor that are distributed in WP](https://github.com/mailpoet/mailpoet/blob/master/lib/AdminPages/Pages/FormEditor.php#L301-L338).

Reply:
🤦🏻 I definitely forgot on translation. I'm sorry.